pkg/bmc/redfish.go

```diff
@@ -78,11 +93,23 @@ func (a *redfishAccessDetails) DriverInfo(bmcCreds Credentials) map[string]inter
 		"redfish_address":   strings.Join(redfishAddress, ""),
 	}
 
+	if a.isiDrac {
+		result["inspect_interface"] = "inspector"
+		result["management_interface"] = "idrac-redfish"
+		result["power_interface"] = "idrac-redfish"
+		result["raid_interface"] = "no-raid"
+		result["vendor_interface"] = "no-vendor"
```

Comment:
Could we turn these into being a part of the interface like BootInterface? The installer will need to make use of these and I had imagined it working like this:

https://github.com/openshift/installer/pull/2766/files#diff-b69d29c6306c28bb385391cd0af1b7fcR69-R75

Reply:
I suppose I could get the info from DriverInfo, but I just wonder why BootInterface is special compared to the other interfaces

Reply:
Also I don't think these should be part of driver info at all,you'll need to specify them like BootInterface is in the provisioner: https://github.com/metal3-io/baremetal-operator/blob/master/pkg/provisioner/ironic/ironic.go#L241

Reply:
Sounds good to me. I just pushed an implementation of that. However, I'm not sure what defaults we should use for the various protocols. I resorted to using empty strings.

Reply:
Good point, Ironic uses default values if unspecified so that's probably ok? We'll need to test.  

It'd be good to get @dhellmann or @zaneb's opinion about this

Reply:
If we do go this route, you'll need to update the linked code above in the provisioner to make use of these new interfaces as well

Reply:
Yeah, I missed this while skim-reading https://docs.openstack.org/ironic/latest/admin/drivers/idrac.html - I think you're correct.

Reply:
`CreateOpts` bits added